Assets/HoloToolkit/SpectatorView/Scripts/Utils/OpenCVUtils.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
+
+using System.Runtime.InteropServices;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.SpectatorView
+{
+	/// <summary>
+	/// Utility function to ensure the OpenCVWrapper has been successfully loaded
+	/// </summary>
+	public class OpenCVUtils : MonoBehaviour
```

Comment:
Probably doesn't need to be a MonoBehaviour.

A static class that's initialized on load before the scene should be sufficient. 

```

```